EDDiscovery/EDDiscoveryController.cs

```diff
@@ -112,8 +112,11 @@ public void Init(bool noreposition)
                 Trace.WriteLine($"Exception: {ex.Message}");
             }
 
+            string proc64 = Environment.Is64BitProcess ? "64-bit" : "32-bit";
+            string os64 = Environment.Is64BitOperatingSystem ? "64-bit" : "32-bit";
             Trace.WriteLine("************************************************");
-            Trace.WriteLine($"EDDiscovery Initializing: {EDDiscovery2.EDDConfig.Options.VersionDisplayString}");
+            Trace.WriteLine($"EDDiscovery Initializing: {EDDiscovery2.EDDConfig.Options.VersionDisplayString} {proc64}");
+            Trace.WriteLine($"OS: {Environment.OSVersion.VersionString} {os64}");
```

Comment:
From Windows 8 onwards (`Microsoft Windows NT 6.2.9200.0`), this will print the lower of either the current operating system version or the highest `supportedOS` in the application manifest.